src/Data/LLVM/BitCode/IR/Constants.hs

```diff
@@ -425,10 +425,14 @@ parseConstantEntry t (getTy,cs) (fromEntry -> Just r) =
         _asmDialect    = mask `shiftR` 2
 
     asmStrSize <- field 1 numeric
+#ifndef QUICK
     Assert.recordSizeGreater r (1 + asmStrSize)
+#endif
```

Comment:
Similarly, rather than surrounding every assertion with CPP, it would be better to redefine each assertion function in terms of `quickMode`. Something like:

```hs
recordSizeGreater x y = Control.Monad.unless quickMode $ ...
```